### ISSUE TITLE
eclipse crashes when a hover starts in a java editor

### ISSUE THREAD

**Issue by jonbarril** [eclipse-platform/eclipse.platform.swt#709]
**Describe the bug**
I start with a clean eclipse install of eclipse (eclipseIDE 20023-06) with only the JDT and subclipse, on Windows 10 (most recent update), I open a java project, open a java source file. I hover over a class. The hover window appears, empty, for a second or two. Eclipse crashes/terminates without warning.

The .meta/log file in the workspace has no stack trace. I have looked around for some sort of crash dump file (after googling for likely spots) but can find none. I tried running eclipse from a cmd window and piping the stdout/stderr output to a file (as described through googling), but that fails, the output file is empty. (It works with windows commands, but not with eclipse.exe).

**To Reproduce**
I open a java project, open a java source file. I hover over a class. The hover window appears, empty, for a second or two. Eclipse crashes/terminates without warning.

**Expected behavior**
The hover should show javadoc and eclipse should not crash.

**Environment:**
1. Select the platform(s) on which the behavior is seen:
- - [] All OS
- - [ x] Windows
- - [ ] Linux
- - [ ] macOS 

2. Additional OS info (e.g. OS version, Linux Desktop, etc)
Windows 10, all recent updates.

4. JRE/JDK version
The JRE that came bundled with the eclipse "Java developer" install. I recall that it is java-17.
 
**Version since**
 eclipse 2023-06 (the very latest).

**Workaround (or) Additional context**
None that I can find.

**Comment by jonbarril** [comment]
In a clean Java project I tried to create a hello world class. When I start typing and auto-suggest pops up a window, eclipse crashes. So, it is just not java type hovering that is crashing eclipse.

Any suggestions on how to capture the crash data?

--jon

**Comment by merks** [comment]
You downloaded a package from here?

https://www.eclipse.org/downloads/packages/

Which comes a JRE.

Or did you use the installer, which lets you choose the JRE. Does the eclipse.ini contain a `-vm`  arguments?

Look for a file like hs_err_pid*.log in your installation (eclipse) folder.

**Comment by mickaelistria** [comment]
Usually, hover crash are symptom of a browser issue (hover uses a web browser under the hood). On Windows IIRC there were some changes (I cannot tell what exactly) on the last stream. Could it be that the browser used by your Eclipse IDE instance is not well supported?

**Comment by Phillipus** [comment]
Blank hovers are usually associated with the in-built Edge browser (`-Dorg.eclipse.swt.browser.DefaultType=edge` in eclipse.ini)

Narrow it down by testing on vanilla Eclipse SDK without any plugins:

https://download.eclipse.org/eclipse/downloads/drops4/R-4.28-202306050440/download.php?dropFile=eclipse-SDK-4.28-win32-x86_64.zip

**Comment by jonbarril** [comment]
Hmmmmmmm. Strange. My windows search bar starting acting up -- try to open, appearing blank, then close. Never saw this behavior before, except recently in eclipse. Restarted windows. Windows came up immediately (no lingering updates) and the search bar started working again. Guess what else started working: Eclipse hovers.

I tried changing eclipse browser setting from internal to system default (Firefox). Restarted eclipse to be sure, and it still works.

I then tried eclipse on my "bad" workspace, which is where this whole trail of tears started, and it seems fine. So, I leave this issue open for a bit longer to be sure but will close it soon if nothing else happens.
